# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/cookiebanners/nsCookieRule.cpp
# Commit: d01a1a055fec
# Full Hash: d01a1a055fece12b906b938954f984bd69278ef7
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2025-04-30 20:49:53
# Regressor Bug: 1960824
# File Overlap Count: 1
# Description:
#   Bug 1960824 - Introduce nsICookie::SAMESITE_UNSET, r=edgul,webdriver-reviewers,extension-reviewers,cookie-reviewers,valentin,Sasha,robwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D246638
# ==============================================================================

diff -r 307ff391dd02 -r d01a1a055fec toolkit/components/cookiebanners/nsCookieRule.cpp
--- a/toolkit/components/cookiebanners/nsCookieRule.cpp	Wed Apr 30 08:53:12 2025 +0000
+++ b/toolkit/components/cookiebanners/nsCookieRule.cpp	Wed Apr 30 08:59:18 2025 +0000
@@ -28,7 +28,7 @@
   net::CookieStruct cookieData(nsCString(aName), nsCString(aValue),
                                nsCString(aHost), nsCString(aPath), 0, 0, 0,
                                aIsHttpOnly, aIsSession, aIsSecure, false,
-                               aSameSite, aSameSite, aSchemeMap);
+                               aSameSite, aSchemeMap);
 
   OriginAttributes attrs;
   mCookie = net::Cookie::Create(cookieData, attrs);